ai-service/src/ai_service/errors.py

```
@@ -0,0 +1,44 @@
+class AIServiceError(Exception):
+    """Base exception for all AI service errors."""
+
+    pass
+
+
+class EmbeddingError(AIServiceError):
+    """Base exception for embedding errors."""
+
+    @classmethod
+    def empty_input(cls):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `empty_input`

(ANN206)